iot-hub/Samples/device/EdgeDownstreamDevice/Properties/launchSettings.json

```diff
@@ -0,0 +1,13 @@
+
+{
+    "profiles": {
+        "EdgeDownstreamDevice": {
+            "commandName": "Project",
+            "environmentVariables": {
+                "DEVICE_CONNECTION_STRING": "[Downstream device IoT Edge connection string]",
+                "EDGE_DEVICE_CA_CERTIFICATE_PATH": "[Path to Edge CA certificate]",
```

Comment:
We provide these as _templates_ to allow unattended build & test during CI. This won't work.